app/src/pages/inside/projectSettingsPageContainer/content/notifications/notifications.jsx

```diff
@@ -160,6 +137,18 @@ export const Notifications = () => {
     );
   };
 
+  useEffect(() => {
+    setHeaderTitleNode(
+      <span className={cx('button')} onClick={addNotificationCase}>
+        <Button disabled={!isAbleToEditNotificationCaseList()}>
+          {formatMessage(messages.create)}
+        </Button>
+      </span>,
+    );
+
+    return () => setHeaderTitleNode(null);
+  });
+
   const onToggleHandler = (isEnabled, notificationCase, id) => {
```

Comment:
Will it be updated in the scope of the next task?

Reply:
Yes, because endpoint for edit is in the next task